README.md

```diff
@@ -130,6 +130,32 @@ GET my-index/doc/1
 
 GET my-index/doc/2
 ```
+During text processing, the plugin clears it. In particular, it cuts out e-mails and links to sites. In addition, the text may contain only numbers and it is impossible to determine the language. In this case, an exception is thrown. This may interfere with bulk queries or reindex queries. A similar situation can be handled like this:
```

Comment:
what/whom do you refer to with 'it' in the first sentence? same applies to the second sentence, maybe you can describe what you mean to make it easier to read.

Reply:
During text processing, the plugin clears it. In particular, langdetect cuts out e-mails and links to sites. In addition, the text can only contain numbers, so langdetect will not be able to determine the language. In this case, an exception is thrown. This may interfere with bulk queries or reindex queries. A similar situation can be handled like this:

Reply:
again, I do not understand what you refer to with `it` here. Is it that the processor removes emails and links. If so, let's write that. Adding a test with a number seems to make sense, I will do that in a separate commit.

Reply:
That's what I meant:
https://github.com/shuyo/language-detection/blob/master/src/com/cybozu/labs/langdetect/Detector.java#L166
language-detection cuts with regular expressions sites and e-mails.
If text content only them then an exception will be thrown because no trigrams will be found.